content/Interfaces/Interfaces.facts.md

```diff
@@ -0,0 +1,69 @@
+### Facts for Interfaces lesson:
+
+An interface is a program that only contains public methods that do not have a method body or implementation. 
+
+An interface defines methods that a class should implement.
+
+The `interface` keyword defines an interface.
+
+To declare an interface, use the `interface` keyword followed by the interface name, and a pair of curly braces `{}` that enclose the definition of all public abstract methods.
+
+The following is an example of an interface declaration:
+
+```php
+interface TestInterface 
+{
+    public function sayHello();
+}
+```
+
+The `implements` keyword lets a class implement an interface.
+
+A class can implement several interfaces. 
+
+A class that implements an interface is required to provide a method body or implementation to all the abstract methods specified in an interface.
```

Comment:
What if: 
A class that implements an interface requires to provide a method body or implementation to all the abstract methods specified in an interface.

Reply:
I would prefer to use the original statement for clarity but if this is something wrong with the grammar then I can rephrase this. Is there something wrong with the original statement? Please check. Thanks.